tests/xdebug_gc_stats1.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+xdebug: No memleak, return empty runs
+--FILE--
```

Comment:
Do the tests work if ```zend.enable_gc ``` is set to 0? If not, either add an ``--INI--`` section to turn them on, or add a ``--SKIPIF--`` (I prefer the ``--INI`` group)